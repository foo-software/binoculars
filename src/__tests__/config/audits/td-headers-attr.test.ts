import auditTdHeadersAttr from '../../../config/audits/td-headers-attr';

describe('td-headers-attr audit', () => {
  it('should match snapshot', () => {
    expect(auditTdHeadersAttr('en')).toMatchSnapshot();
  });
});
