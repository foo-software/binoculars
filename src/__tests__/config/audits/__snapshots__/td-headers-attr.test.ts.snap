// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`td-headers-attr audit should match snapshot 1`] = `
Object {
  "description": "Screen readers have features to make navigating tables easier. Ensuring \`<td>\` cells using the \`[headers]\` attribute only refer to other cells in the same table may improve the experience for screen reader users. [Learn more](https://web.dev/td-headers-attr/).",
  "title": "Cells in a \`<table>\` element that use the \`[headers]\` attribute refer to table cells within the same table.",
}
`;
